packages/create-cloudflare/templates-experimental/react/js/wrangler.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "<TBD>",
+	"main": "api/index.js",
```

Comment:
indentation here is odd 🙃 . no biggie just calling it out. prolly prettier will fix, if you haven't run it already